### DISCUSSION TITLE
https://morrowind-modding.github.io/guides/interiors/morrowind-interiors-project

### DISCUSSION THREAD

**Discussion by giscus** [morrowind-modding/morrowind-modding.github.io#102] [General]
# https://morrowind-modding.github.io/guides/interiors/morrowind-interiors-project

Fake exterior worldspaces using Export Sphere and Blender.

https://morrowind-modding.github.io/guides/interiors/morrowind-interiors-project

**Comment by magicaldave** [comment]
Aw jeez rick you totally fucked it
